content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -59,7 +59,7 @@ Resources consumed by the command are counted against the Container.
   This is a beta-level feature default enabled by the `PodLifecycleSleepAction` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/). 
```

Comment:
Is this true? Do we still need the note callout block / can we simplify?

Reply:
The feature gate would be enabled by default. We can remove the callout if that makes sense and mention that Sleep action supports a value of zero from v1.33. I guess this would be okay since sleep action also is at beta stage right now and is not graduating to GA in v1.33.

Reply:
I misunderstood the comment initially. The PodLifecycleSleepAction feature gate is still in beta. Since both `PodLifecycleSleepAction` and `PodLifecycleSleepActionAllowZero` feature gates are in beta and enabled by default, we can possibly simplify the callout block. Shall I add the line about the zero value in along with the description for sleep in line 59?